src/interface/guide/components/GuideDivs.tsx

```diff
@@ -67,3 +67,252 @@ export const PanelHeader = styled.div`
     height: 24px;
   }
 `;
+
+// ==================
+// Shared Components for Guide Sections with Stats
+// ==================
+
+/** Container for entire component section */
+export const SectionContainer = styled.div`
+  margin-bottom: 16px;
+`;
+
+/** Header section containing title and stats */
+export const SectionHeader = styled.div`
+  display: flex;
+  gap: 16px;
+  align-items: stretch;
+  margin-bottom: 8px;
+`;
+
+/** Column for title and labels */
+export const TitleColumn = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  gap: 4px;
+  justify-content: space-between;
+`;
+
+/** Main title/header for guide sections */
+export const SectionTitle = styled.h3`
+  margin: 0;
+  font-size: 18px;
+  font-weight: 600;
+  color: #fab700;
+  text-shadow: 0 2px 4px rgba(0, 0, 0, 0.5);
+`;
+
+/** Small label text (e.g., "Timeline", "Performance") */
+export const Label = styled.div`
+  font-size: 11px;
+  font-weight: 600;
+  color: rgba(255, 255, 255, 0.6);
+  text-transform: uppercase;
+  letter-spacing: 1px;
+`;
+
+/** Column container for stat cards and helper text */
+export const StatColumn = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-end;
+`;
+
+/** Row for stat cards */
+export const StatsRow = styled.div`
+  display: flex;
+  gap: 8px;
+  align-items: flex-end;
+`;
+
+/** Individual stat card with colored border */
+export const StatCard = styled.div<{ color: string }>`
+  background: rgba(0, 0, 0, 0.3);
+  border-radius: 6px;
+  padding: 6px 12px;
+  min-width: 70px;
+  border-left: 3px solid ${(props) => props.color};
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.2);
+`;
+
+/** Stat value (large number/text) */
+export const StatValue = styled.div`
+  font-size: 18px;
+  font-weight: 700;
+  color: #fff;
+  line-height: 1;
+  margin-bottom: 2px;
+  text-shadow: 0 1px 3px rgba(0, 0, 0, 0.5);
+`;
+
+/** Stat label (small descriptive text) */
+export const StatLabel = styled.div`
+  font-size: 10px;
```

Comment:
10px is generally too small. i know we have some existing uses of fonts this small, but we probably shouldn't in most cases.

as an additional note: prefer scalable units (like `rem`) instead of fixed-size units (like `px`) because they will be handled better by browsers on more combinations of screen sizes / dpis. (i actually get some minor pixelation / artifacting at 10px on my hidpi work monitor)

i know a bunch of these were a part of a previous PR and I'm not going to demand changing all of them right now. however, please adjust the 10px and 11px fonts up to at least 12px (and preferably a little larger, like 14px, if it fits in the UI)